tools/dumpdfs/dumpdfs.c

```diff
@@ -5,6 +5,12 @@
 #include "dragonfs.h"
 #include "dfsinternal.h"
 
+#if BYTE_ORDER == BIG_ENDIAN
```

Comment:
I know why this didn't work. I started developing this toolchain on sparc so of course it wouldn't work on intel.